src/pages/dataset-visualization-view/dataset-visualization-view.scss

```diff
@@ -8,12 +8,13 @@ main-app-element {
 dataset-visualization {
   .visualization-header {
     .header {
-      display: flex;
+      display: flow-root;
       justify-content: space-between;
     }
 
     .button {
       @include button();
+      float: right;
```

Comment:
I'm not a huge fan of float (it's more unpredictable/uncontrollable than flex), but if it helps make plotly not buggy then it's fine